### ISSUE TITLE
请问有demo吗？demo怎么使用？

### ISSUE THREAD

**Issue by hejunv** [AriaLyy/Aria#850]
请问有简单的可以运行的demo吗？
全部下载也运行不了啊。
## 版本
* 框架版本

* 系统版本

## 错误日志

Execution failed for task ':Aria:processDebugManifest'.
> Cannot read packageName from /home/android/AndroidStudioProjects/Aria-master/Aria/src/main/AndroidManifest.xml

## 重现步骤

1.下载zip
2.导入android studio
3.选中app，运行
4.报错，没有主配置文件

**Comment by hejunv** [comment]
https://www.jianshu.com/p/ba32dc499db3 《一行代码实现android文件下载》这个博客给的代码进过移植可以运行耶。解决我的问题了。

**Comment by zhengxinghua1101** [comment]
居然在这里找到你了。

**Comment by hejunv** [comment]
> 居然在这里找到你了。

不是我给你讲这个，又怎会找到，哈哈哈哈
